src/dialog/styles/dialog.m.css

```diff
@@ -11,7 +11,7 @@
 	width: 450px;
 	overflow: hidden;
 	max-width: calc(100% - 30px);
-	z-index: 2;
+	z-index: calc(var(--zindex-dialog) + 1);
```

Comment:
Why is the `+ 1` necessary? Shouldn't the dialog use the `--zindex-dailog` CSS variable directly?